mungegithub/mungers/approval-handler.go

```diff
@@ -64,6 +68,23 @@ func (*ApprovalHandler) EachLoop() error { return nil }
 // AddFlags will add any request flags to the cobra `cmd`
 func (*ApprovalHandler) AddFlags(cmd *cobra.Command, config *github.Config) {}
 
+// Returns associated issue, or 0 if it can't find any.
+// This is really simple, and could be improved later.
+func findAssociatedIssue(body *string) int {
+	if body == nil {
+		return 0
+	}
+	match := regexp.MustCompile(`#(\d+)`).FindStringSubmatch(*body)
```

Comment:
> By the way, do we need to take care of situation like abc#1234abc?

We're trying to keep it as simple as possible for the moment

> Will FindString suffice?

Sure, I could use `FindString` and then manually remove the pound sign. Doesn't make much difference to me.